ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/EquipmentUpgradePlannerViewModel.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using CommunityToolkit.Mvvm.Input;
+using ElectronicObserver.Common;
+using ElectronicObserver.Services;
+using ElectronicObserverTypes;
+
+namespace ElectronicObserver.Window.Tools.EquipmentUpgradePlanner;
+
+public partial class EquipmentUpgradePlannerViewModel : WindowViewModelBase
+{
+	private ObservableCollection<EquipmentUpgradePlanItemViewModel> PlannedUpgrades { get; set; } = new();
```

Comment:
I don't think set is needed for these 2 collections.

Reply:
Yeah, set on an observablecollection doesn't make much sense too